checkov/terraform/checks/graph_checks/aws/CloudFrontS3OriginConfigWithOAI.yaml

```diff
@@ -19,12 +19,11 @@ definition:
     - cond_type: attribute
       resource_types:
         - aws_cloudfront_distribution
-      attribute: "origin.*.s3_origin_config"
+      attribute: "origin.*.s3_origin_config.origin_access_identity"
       operator: "exists"
```

Comment:
The more I think about it: this probably should check "not empty", not just "exists". 

Same below. 

WDYT?

Reply:
you can change it to a 'not empty', but a not empty `origin_access_identity` is a broken configuration, but correct me, if I'm wrong.